play/src/pages/PhaseResults.tsx

```diff
@@ -82,44 +82,44 @@ export default function PhaseResults({
     gameSession.currentState
   );
 
-  const [newPoints, setNewPoints] = React.useState<number>(0);
-  // update teamscore on the backend, if it fails, flag the error to pop the error modal
-  const updateTeamScore = async (inputTeamId: string, newScore: number) => {
-    try {
-      await apiClients.team.updateTeam({ id: inputTeamId, score: newScore + score });
-      setNewPoints(newScore);
-    } catch {
-      setIsError({ error: true, withheldPoints: newScore });
-    }
-  };
+  // const [newPoints, setNewPoints] = React.useState<number>(0);
```

Comment:
I think we can just go ahead and delete this page if we don't need it

Reply:
deleted